lib/menu.rb

```diff
@@ -0,0 +1,32 @@
+require 'yaml'
+
+class Menu
+
+  # DEF_MENU = [{dish: 'fries', price: 1.0}, {dish: 'fish', price: 1.5}]
+
+  def initialize(file_name = 'menu.yml')
+    @menu = YAML.load_file(file_name)
+    # @menu = YAML::load File.read menu
+    # @menu = YAML.load(File.open("menu.yml"))
+  end
+
+  def print
+    k=1
+    @menu.each do |item|
+      puts "#{k}. #{item[:dish]} ... £#{'%.2f' % item[:price]} "
+      k+=1
+    end
+  end
+
+  def dishes
+    @menu
```

Comment:
You're exposing the internal array here.  At the very least use `dup`.  Otherwise add the relevant methods to `Menu` so I never need access to the array.